public/locales/en/translation.json

```diff
@@ -303,5 +303,26 @@
     "contactUsModal": {
       "message": "Thank you for leaving us message!"
     }
+  },
+  "signIn": {
+    "header": "Welcome Back!",
+    "SignInWith": "Sign in with",
+    "newHere": "New here? ",
+    "createAccount": "Create an account",
+    "form": {
+      "yourEmail":"Your Email",
+      "yourPassword":"Your password",
+      "rememberMe":"Remember me next time",
+      "signIn":"Sign in"
+    },
+    "validationMessage": {
+      "incorrectPassword":"The password that you've entered is incorrect",
+      "emailNotFound":"The email address that you've entered doesn't match any account. Sign up for an account.",
```

Comment:
Sign in or sign up? I think it should be sign in.

Reply:
sign up, as he is trying to sign in before creating an account first

Reply:
so, logically if I am trying trying to sign in and I get "The email address that you've entered doesn't match any account" this message, it means that I should try to sign in again or sign up. Or you should write "The email address that you've entered doesn't not exist. Sign up for an account."

Reply:
I got that message from Facebook. They say the same. The reason behind that might be that an account might be linked with more than one email.